tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -85,6 +88,22 @@ pub struct Ciphertext {
     pub pbs_order: PBSOrder,
 }
 
+impl crate::Named for Ciphertext {
+    const NAME: &'static str = "shortint::Ciphertext";
+}
+
+impl ParameterSetConformant for Ciphertext {
+    type ParameterSet = CiphertextConformanceParams;
+
+    fn is_conformant(&self, param: &CiphertextConformanceParams) -> bool {
+        self.ct.as_view().is_conformant(&param.ct_params)
+            && param.message_modulus == self.message_modulus
+            && param.carry_modulus == self.carry_modulus
+            && param.pbs_order == self.pbs_order
+            && self.degree.0 == self.message_modulus.0 - 1
```

Comment:
param on the right please to avoid mistakes (not self) for the degree check

also maybe have all self accesses on the left ? or all params accesses, but I would like to avoid mixing

Reply:
Done